cmd/server/main.go

```diff
@@ -207,29 +206,15 @@ func validateNetworkName(cfg *config, cChainClient client.Client) error {
 		if cfg.NetworkName != "" {
 			return fmt.Errorf("network name is not provided, can't fetch network name in offline mode")
```

Comment:
I don't think we can do that. We are expected to inject network name in https://github.com/ava-labs/avalanche-rosetta/blob/get_network_name_fix/cmd/server/main.go#L120-L130. 

In case of offline mode, I think we should trust the user to set the correct network name in the config.

Reply:
So I understand you're say we always want NetworkName configured, whether it's online or offline mode.
As such I moved the network name validation to the cfg.validate method, where all "static" checks are carried out.